core-bundle/contao/themes/flexible/styles/main.css

```diff
@@ -2772,9 +2772,17 @@ fieldset.tl_tbox > legend, fieldset.tl_box > legend {
 	color: var(--legend);
 	padding: 9px 12px 9px 28px;
 	background: url("icons/navcol.svg") 13px 10px no-repeat;
+}
+
+fieldset.tl_tbox > legend[onclick], fieldset.tl_box > legend[onclick] {
 	cursor: pointer;
 }
```

Comment:
This one isn't needed since the `onclick` is replaced by a stimulus controller in Contao 5.3, hence you wouldn't even need to match the `button`. We could simply leave it at the legend (even if it's the same cursor as the button)
```suggestion
	cursor: pointer;
}
```

Reply:
It is needed - this acts as a fallback for when you don't have the `<button>` but still use the `onclick` handler on the `<legend>`.

Reply:
I moved it back to the `legend` as you can see. It will still be "clickable"

Reply:
But that is not correct. You should not use `cursor: pointer` on elements that actually do nothing. This is likely also why #7526 went unnoticed so far, as `cursor: pointer` was on the `<legend>` instead of the `<button>`, giving no immediate visual indication that the _actual_ clickable area was smaller than it appeared to be.

Also you need to consider the following three cases:

```html
<fieldset class="tl_box">
  <legend>Foobar</legend>
```
```html
<fieldset class="tl_box">
  <legend onclick="AjaxRequest.toggleFieldset(…)">Foobar</legend>
```
```html
<fieldset class="tl_box">
  <legend><button type="button" data-action="contao--toggle-fieldset#toggle" aria-expanded="true">Foobar</button></legend>
```

In the first case `cursor: pointer` must not be added to the `<legend>`. The current state of this PR would reflect that.

Reply:
This should most likely be a bugfix in 4.13 as well if that's the case but it's unrelated to this PR rn